contracts/modules/staking/PooledStaking.sol

```diff
@@ -737,47 +957,519 @@ contract PooledStaking is IPooledStaking, MasterAware {
     }
   }
 
-  function migrateToNewV2Pool(address stakerAddress) external {
+  function getV1ProceForProduct(uint id) pure internal returns (uint) {
+    // {V1_PRICES_HELPER_BEGIN}
+
+    // bZx v1
+    if (id == 0) {
+      return 42880894339275514000; // 42.880894339275514%
+    }
+
+    // Saturn DAO Token
+    if (id == 1) {
+      return 63420369661816350000; // 63.42036966181635%
+    }
+
+    // Legacy Gnosis MultiSig
+    if (id == 2) {
+      return 25075515385886447000; // 25.075515385886444%
+    }
+
+    // dxDAO
+    if (id == 3) {
+      return 24184951105233400000; // 24.1849511052334%
+    }
+
+    if (
+      // Argent
+      id == 4 ||
+      // dydx Perpetual
+      id == 5 ||
+      // Compound v2
+      id == 11 ||
+      // Gnosis Safe
+      id == 12 ||
+      // MakerDAO MCD
+      id == 14 ||
+      // Yearn Finance (all vaults)
+      id == 18 ||
+      // Curve All Pools (incl staking)
+      id == 21 ||
+      // Uniswap v2
+      id == 23 ||
+      // mStable
+      id == 29 ||
+      // Synthetix
+      id == 30 ||
+      // Bancor v2
+      id == 33 ||
+      // UMA
+      id == 34 ||
+      // Set Protocol v2
+      id == 43 ||
+      // Eth 2.0 (deposit contract)
+      id == 45 ||
+      // Keeper DAO
+      id == 47 ||
+      // Aave v2
+      id == 51 ||
+      // SushiSwap v1
+      id == 52 ||
+      // BadgerDAO
+      id == 54 ||
+      // Reflexer
+      id == 58 ||
+      // Stake DAO
+      id == 61 ||
+      // Liquity
+      id == 62 ||
+      // Uniswap v3
+      id == 64 ||
+      // Barnbridge Smart Yield v1
+      id == 65 ||
+      // Convex Finance v1
+      id == 66 ||
+      // Alpha Homora v2
+      id == 68 ||
+      // Balancer v2
+      id == 69 ||
+      // BlockFi
+      id == 74 ||
+      // Nexo
+      id == 75 ||
+      // Ledn
+      id == 77 ||
+      // Hodlnaut
+      id == 78 ||
+      // Binance
+      id == 79 ||
+      // Coinbase
+      id == 80 ||
+      // Kraken
+      id == 81 ||
+      // Gemini
+      id == 82 ||
+      // FTX
+      id == 83 ||
+      // Crypto.com
+      id == 84 ||
+      // Yield.app
+      id == 85 ||
+      // Rari Capital
+      id == 88 ||
+      // Abracadabra
+      id == 89 ||
+      // Anchor
+      id == 91 ||
+      // Yearn yvUSDC v2
+      id == 97 ||
+      // Yearn ycrvstETH v2
+      id == 98 ||
+      // Curve 3pool LP (3Crv)
+      id == 99 ||
+      // Convex stethCrv (cvxstethCrv)
+      id == 103 ||
+      // Convex 3CRV (cvx3CRV)
+      id == 104 ||
+      // Convex mimCrv (cvxmimCrv)
+      id == 105 ||
+      // Popsicle Finance
+      id == 106 ||
+      // Notional Finance v2
+      id == 107 ||
+      // OlympusDAO
+      id == 108 ||
+      // Ribbon Finance v2
+      id == 109
+    ) {
+      return 2600000000000000000; // 2.6%
+    }
+
+    // DDEX
+    if (id == 6) {
+      return 56398373648265250000; // 56.39837364826525%
```

Comment:
are we going to keep updating these to match the latest number?

Reply:
Yes, `get-v1-cover-prices.js` does that automatically.